.github/actions/python/action.yaml

```diff
@@ -8,12 +8,24 @@ inputs:
 runs:
   using: "composite"
   steps:
+
     - name: Set up Python 3.9 for protos
+      uses: useblacksmith/setup-python@v6
+      # NOTE(hammadb) Blacksmith does not support windows, so we use the official action for windows and
+      # skip it on other platforms
+      if: runner.os != 'Windows'
+      with:
+        python-version: "3.9"
```

Comment:
should this use the `python-version` input instead of hard-coding

Reply:
This is just how it was. I don't view this as a part of the scope of these changes.